cmd/root.go

```diff
@@ -162,6 +173,31 @@ func getCacheDir(imageName string) (string, error) {
 	return filepath.Join(rootDir, filepath.Clean(imageName)), nil
 }
 
+func getWriter(outputFile string) (io.Writer, error) {
+	var err error
+	var outWriter io.Writer
+
+	// If the user specifies an output file, ensure exists
+	if outputFile != "" {
+
+		// Don't overwrite a file that exists, unless given --force
+		if _, err := os.Stat(outputFile); !os.IsNotExist(err) {
```

Comment:
you can combine the next check into this line:

```golang
if _, err := os.Stat(outputFile); !os.IsNotExist(err) && !forceWrite {
  ...
}
```

Reply:
oh yes, definitely like that better.